src/Files.App/Program.cs

```diff
@@ -17,12 +17,16 @@ namespace Files.App
 {
     internal class Program
     {
+        [DllImport("Microsoft.ui.xaml.dll")]
+        private static extern void XamlCheckProcessRequirements();
+
         // Note: We can't declare Main to be async because in a WinUI app
         // this prevents Narrator from reading XAML elements.
         // https://github.com/microsoft/WindowsAppSDK-Samples/blob/main/Samples/AppLifecycle/Instancing/cs-winui-packaged/CsWinUiDesktopInstancing/CsWinUiDesktopInstancing/Program.cs
         [STAThread] // STAThread has no effect if main is async, needed for Clipboard
         private static void Main()
         {
+            XamlCheckProcessRequirements();
```

Comment:
What does this method do?

Reply:
Found in App.g.cs from a newly created WinUI 3 app.

Reply:
We should be able to remove, app.g.cs is auto generated and will take care if this if needed.